package.json

```diff
@@ -26,6 +32,7 @@
     "@babel/preset-typescript": "^7.13.0",
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
+    "rimraf": "^3.0.2",
     "typescript": "^4.2.4"
   },
   "peerDependencies": {
```

Comment:
You might want to add `"files": ["dist"]` to limit the files pushed to npm to be `dist`